### PR TITLE
Event managers should be able to add staffs while creating an event

### PR DESCRIPTION
**Pull request by andela-ooranagwa**
Why?
Part of the information required for an event include the staffs for
the event. As such, being able to add this information while creating an
event _kind of_ unifies the process and should improve user experience.

This commit address this by:
* Adding accepted accept nested attributes feature to event to creation
* form
* Providing input field for managers to enter staff email when creating
* events
* Implementing Jquery autocomplete to enable managers add more than one
* staff at time
* Creating partial views for ajax query result for email autocomplete
* request

Finishes [#111956121]

### CODE REVIEW

File: app/decorators/event_decorator.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@ def event_template
   end
 
   def end_date
-    object.end_date ? object.end_date.strftime("%b %d %Y") : ""
+    object.end_date ? object.end_date.strftime("%b %d %Y") : Time.zone.now
+  end
+
+  def get_event_staffs
```
Comment (andela-iokonkwo) [inline app/decorators/event_decorator.rb]: Love what you did here. :+1:

File: app/views/users/_fetch_user_info.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="chip">
```
Comment (andela-iokonkwo) [inline app/views/users/_fetch_user_info.html.erb]: The role passed when this partial is called is not been displayed here. Why is that?

**Comment by andela-ooranagwa** [comment]
That's an oversight. I've made the correction.

**Comment by andela-ooranagwa** [comment]
Yes, you are right. But I wanted the display to be something like:
`Username (role)` instead of `Username role` or `Username ()` (when no role has been assigned like when an event is created)

**Comment by andela-iokonkwo** [comment]
You handled the `()` part in the decorator.

**Comment by andela-ooranagwa** [comment]
Oh! I see now. I fix it at once. Thanks.

**Comment by andela-iokonkwo** [comment]
:white_check_mark: Good Job

**Pull request merged**